components/remote_settings/src/jexl_filter.rs

```diff
@@ -0,0 +1,52 @@
+use crate::RemoteSettingsContext;
+use firefox_versioning::compare::version_compare;
+use jexl_eval::Evaluator;
+use serde_json::{json, Value};
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub(crate) enum ParseError {
+    #[error("Evaluation error: {0}")]
+    EvaluationError(String),
+    #[error("Invalid result type")]
+    InvalidResultType,
+}
+
+pub struct JexlFilter {
+    evaluator: Evaluator<'static>,
+    context: Value,
+}
+
+impl JexlFilter {
+    pub(crate) fn new(context: Option<RemoteSettingsContext>) -> Self {
+        let env_context = match context {
+            Some(ctx) => {
+                serde_json::to_value(ctx).expect("Failed to serialize RemoteSettingsContext")
+            }
+            None => json!({}),
+        };
+
+        Self {
+            evaluator: Evaluator::new()
+                .with_transform("versionCompare", |args| Ok(version_compare(args)?)),
```

Comment:
Please a comment here (or open a follow-up issue) that eventually we want to support the same transports as desktop https://remote-settings.readthedocs.io/en/latest/target-filters.html#transforms